test/weights.jl

```diff
@@ -2,6 +2,8 @@ using StatsBase
 using LinearAlgebra, Random, SparseArrays, Test
 
 @testset "StatsBase.Weights" begin
+# NOTE: Do not add eweights here, as its methods don't match those of the others, so the
```

Comment:
I think this comment isn't needed: it's clear `eweights` is different, if only because it doesn't have a corresponding type.

Reply:
Okay